src/P5Functions/Math.js

```diff
@@ -426,13 +426,20 @@ module.exports = function withMath(p, undef) {
   * @see noise
   */
   p.random = function() {
-    if(arguments.length === 0) {
-      return internalRandomGenerator();
-    }
-    if(arguments.length === 1) {
-      return internalRandomGenerator() * arguments[0];
+    var aMin, aMax;
+    switch (arguments.length) {
```

Comment:
Switches are great for long chains, but in this case we can use two `if` statements instead:
```
p.random = function(aMin, aMax) {
  if (arguments.length === 0) {
    aMin = 0;
    aMax = 1;
  } else if (arguments.length === 1) {
    aMax = aMin;
    aMin = 0
  }
  ...
```